src/genericsite/templates/genericsite/base.html

```diff
@@ -17,21 +17,49 @@
   crossorigin="anonymous">
   {% endblock bootstrap_styles %}
   {% block opengraph %}{{ opengraph }}{% endblock %}
-  {% block extra_head %}{% endblock %}
+  {% block extra_head %}
+  <link rel="stylesheet" href="{% static 'generic.css' %}">
+  {% endblock %}
 </head>
 <body class="{% block body_class %}{% endblock body_class %}">
-  {% block header %}{% include header_template|default:"genericsite/blocks/header_simple.html" %}{% endblock header %}
-  {% block precontent %}{% include precontent_template|default:"genericsite/blocks/empty.html" %}{% endblock precontent %}
-  {% block content %}{% include content_template|default:"genericsite/blocks/article_text.html" %}{% endblock content %}
-  {% block postcontent %}{% include postcontent_template|default:"genericsite/blocks/empty.html" %}{% endblock postcontent %}
-  {% block footer %}{% include footer_template|default:"genericsite/blocks/footer_simple.html" %}{% endblock footer %}
+  {% block header %}
+  <header class="{{ bootstrap_container_class }}">
```

Comment:
Pulling the container class out of the other templates and putting them in this base template makes a lot of sense. +1